db/migrate/20231027144605_add_unit_amount_to_fees.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class AddUnitAmountToFees < ActiveRecord::Migration[7.0]
+  def up
+    change_column :fees, :unit_amount_cents, :bigint, null: false, default: 0
+    add_column :fees, :unit_amount, :decimal, precision: 30, scale: 5, null: false, default: '0.0'
+
+    execute <<-SQL
+      UPDATE fees
+      SET unit_amount = unit_amount_cents / 100.0
+      WHERE unit_amount_cents != 0;
+    SQL
+  end
+
+  def down
+    change_column :fees, :unit_amount_cents, :decimal, precision: 30, scale: 5, null: false
```

Comment:
Rails/BulkChangeTable: You can use `change_table :fees, bulk: true` to combine alter queries. (https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_table, https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html)